src/libraries/System.Net.Http/src/System/Net/Http/HeaderEncodingSelector.cs

```diff
@@ -10,6 +10,7 @@ namespace System.Net.Http
     /// </summary>
     /// <param name="headerName">Name of the header to specify the <see cref="Encoding"/> for.</param>
     /// <param name="context">The <typeparamref name="TContext"/> we are enoding/decoding the headers for.</param>
```

Comment:
There's a typo one line above, could you please fix that as well:
```suggestion
    /// <param name="context">The <typeparamref name="TContext"/> we are encoding/decoding the headers for.</param>
```